README.md

```diff
@@ -64,3 +64,15 @@ Serving web UI on http://localhost:57599
 <img width=500px src="docs/graph.png" alt="graph"/>
 
 <img width=500px src="docs/flamegraph.png" alt="flamegraph"/>
+
+
+## IDE Integration
+### VSCode
+If running  `pprof` in VSCode integrated terminal and getting the following error in the Web View:
+```
+Could not execute dot; may need to install graphviz.
+``` 
+
+You can try the following:
+1. Install  [`graphviz`](https://graphviz.org/) on your local machine
```

Comment:
Maybe split this into two sections? The first one about the fact that you can use the Go Profiling extension together with this package?
That's neat and I didn't know that.

Secondly if one comes across the error that `dot` is not available then graphviz should be installed.

Reply:
Hey! Thanks for responding so quickly. I'll look more into it!